web/packages/core/src/js-polyfills.ts

```diff
@@ -3,6 +3,7 @@ declare global {
         Prototype?: {
             Version?: string;
         };
+        Map: typeof Map;
```

Comment:
`Map` of course exists in the global namespace. That inherently makes it available on Window, but alas TypeScript does not see it that way.